### PR TITLE
Sync release notes from getambassador.io.git

### PR DESCRIPTION
**Pull request by kflynn**
We made edits to the release notes over in getambassador.io.git; the changes need to be here, too.

Signed-off-by: Flynn <flynn@datawire.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -139,24 +144,26 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
 
 ### Emissary-ingress and Ambassador Edge Stack
 
-- Feature: It is now possible to use both the `getambassador.io/v2` and `getambassador.io/v3alpha1`
-  apiVersions.  This is accomplished with the addition of an `emissary-ingress-apiext` Service and
-  Deployment that handles converting between the two versions for the apiserver.  This allows the
-  user to semlessly request and author resources in whichever API version is desired, and
-  facilitates easier migration from Emissary 1.x to 2.x.  Resources authored in v3alpha1 will not be
-  visible to Emissary 1.x; resources authored in v2 will be visible to both Emissary 1.x and 2.x.
+- Change: Support for the Envoy V2 API is deprecated in Emissary-ingress v2.1, and will be removed
```
Comment (AliceProxy) [inline CHANGELOG.md:147]: Is it intentional that we have this message in two places?

File: CHANGELOG.md [hunk 2]
```diff
@@ -139,24 +144,26 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
...
+  in Emissary-ingress v2.2.0. The `AMBASSADOR_ENVOY_API_VERSION` environment variable will be
+  removed at the same time. Only the Envoy V3 API will be supported (this has been the default since
+  Emissary-ingress v1.14.0).
+
+- Feature: Emissary-ingress supports `getambassador.io/v2` CRDs, to simplify migration from
+  Emissary-ingress 1.X. **Note:** it is important to read the <a
+  href="../topics/install/migration-matrix">migration documentation</a> before starting migration.
```
Comment (AliceProxy) [inline CHANGELOG.md:154]: This takes you to https://github.com/emissary-ingress/emissary/blob/topics/install/migration-matrix which does not exist.

File: CHANGELOG.md [hunk 3]
```diff
@@ -322,8 +329,8 @@ href="https://a8r.io/slack">Slack</a> and let us know what you think.
   configuration changes that are not backwards compatible with the 1.X family.  API versions
   `getambassador.io/v0`, `getambassador.io/v1`, and `getambassador.io/v2` are deprecated.  Further
   details are available in the <a
-  href="../about/changes-2.0.0/#1-configuration-api-version-xgetambassadoriov3alpha1">2.0.0
-  Changes</a> document.
+  href="../about/changes-2.x/#1-configuration-api-version-getambassadoriov3alpha1">Major Changes in
+  2.X</a> document.
```
Comment (AliceProxy) [inline CHANGELOG.md:333]: same issue with this link, etc.

File: docs/releaseNotes.yml [hunk 4]
```diff
@@ -146,16 +160,21 @@ items:
         body: >-
           <i>Emissary-ingress 2.1.0 is not recommended; upgrade to 2.1.2 instead.</i>
 
-      - title: CRD conversions
+      - title: Envoy V2 API deprecation
```
Comment (AliceProxy) [inline docs/releaseNotes.yml:163]: Same question as I had above about if we mean to have this entry in two places

File: CHANGELOG.md (hunk 1 above)
Comment (kflynn) [reply CHANGELOG.md:147]: Yes, the first is for 2.1.2 and this one is for 2.1.0.

File: CHANGELOG.md (hunk 2 above)
Comment (LukeShu) [reply CHANGELOG.md:154]: lmao

Comment (LukeShu) [reply CHANGELOG.md:154]:
```patch
diff --git a/docs/CHANGELOG.tpl b/docs/CHANGELOG.tpl
index 5f82b2329..999ea5a15 100644
--- a/docs/CHANGELOG.tpl
+++ b/docs/CHANGELOG.tpl
@@ -106,6 +106,7 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
     strings.ReplaceAll "</i>" "*" |
     strings.ReplaceAll "<code>" "`" |
     strings.ReplaceAll "</code>" "`" |
+    strings.ReplaceAll "href=\"../" "href=\"https://www.getambassador.io/docs/emissary/latest/" |
     strings.WordWrap 100 }}
 {{- end }}{{ end }}{{ end }}
 {{ if ne $release.date "N/A" }}
@@ -119,6 +120,7 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
     strings.ReplaceAll "</i>" "*" |
     strings.ReplaceAll "<code>" "`" |
     strings.ReplaceAll "</code>" "`" |
+    strings.ReplaceAll "href=\"../" "href=\"https://www.getambassador.io/docs/emissary/latest/" |
     strings.WordWrap 98 |
     strings.Indent 2 |
     strings.TrimPrefix "  " }}{{ if index . "github" }}{{ range .github }} ([{{.title}}]){{ end }}{{ end }}
@@ -135,6 +137,7 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
     strings.ReplaceAll "</i>" "*" |
     strings.ReplaceAll "<code>" "`" |
     strings.ReplaceAll "</code>" "`" |
+    strings.ReplaceAll "href=\"../" "href=\"https://www.getambassador.io/docs/edge-stack/latest/" |
     strings.WordWrap 98 |
     strings.Indent 2 |
     strings.TrimPrefix "  " }}{{ if index . "github" }}{{ range .github }} ([{{.title}}]){{ end }}{{ end }}
```
aught to do it

**Review by LukeShu — CHANGES_REQUESTED** [review]
`CHANGELOG.tpl` needs fixed to handle links

**Review by LukeShu — APPROVED** [review]

**Pull request merged**